benchmarks/test_benchmark_coo.py

```diff
@@ -15,16 +15,19 @@ def format_id(format):
 
 
 @pytest.mark.parametrize("format", ["coo", "gcxs"])
-def test_matmul(benchmark, sides, format, seed, max_size, ids=format_id):
+def test_matmul(benchmark, sides, seed, format, backend, min_size, max_size, ids=format_id):
     m, n, p = sides
 
-    if m * n >= max_size or n * p >= max_size:
+    if m * n >= max_size or n * p >= max_size or m * n <= min_size or n * p <= min_size:
         pytest.skip()
 
     rng = np.random.default_rng(seed=seed)
     x = sparse.random((m, n), density=DENSITY, format=format, random_state=rng)
     y = sparse.random((n, p), density=DENSITY, format=format, random_state=rng)
 
```

Comment:
All the tests on file `test_benchmark_coo.py` were meant to fail with `Finch`. 
`Finch` `sparse.random` (used to build  `x` & `y`, line 25 and 26) is different, because it doesn't have the `format` argument that is used with `Numba`.

Reply:
@mtsokol How hard would it be to add this to `finch`?

Reply:
You can probably use `sparse.asarray(..., format=<format>)`

Reply:
Do we just need a format=nothing default and then a reformatting line inside finch.random?

Reply:
That'd suffice -- if the format isn't what the `format` arg demands, reformat.

Reply:
do we have a way of specifying format currently?